python/docs/source/getting_started/install.rst

```diff
@@ -157,7 +157,7 @@ Package                    Supported version Note
 ========================== ========================= ======================================================================================
 `py4j`                     >=0.10.9.7                Required
 `pandas`                   >=1.4.4                   Required for pandas API on Spark and Spark Connect; Optional for Spark SQL
-`pyarrow`                  >=4.0.0                   Required for pandas API on Spark and Spark Connect; Optional for Spark SQL
+`pyarrow`                  >=10.0.1                  Required for pandas API on Spark and Spark Connect; Optional for Spark SQL
```

Comment:
Can we don't change this upperbound though?

Reply:
I am asking to keep `10.0.1` only at [dev/create-release/spark-rm/Dockerfile](https://github.com/apache/spark/pull/45892/files#diff-74d65c5e02efda4c0b26cb1d6f8b8dd08300546e5ceac4253d1428475ec206f6)

Reply:
Ah, I see

Reply:
Updated!